nixos/tests/wasabibackend.nix

```diff
@@ -30,9 +30,5 @@ import ./make-test-python.nix ({ pkgs, ... }: {
     machine.wait_until_succeeds(
         "grep 'Wasabi Backend started' /var/lib/wasabibackend/.walletwasabi/backend/Logs.txt"
     )
-    machine.sleep(5)
-    machine.succeed(
-        "grep 'Config is successfully initialized' /var/lib/wasabibackend/.walletwasabi/backend/Logs.txt"
-    )
```

Comment:
The reason for removing this is it got removed [removed](https://github.com/WalletWasabi/WalletWasabi/commit/e95b4ff59350d1cf93e0c2c110147e52af63c0bb#diff-31196a68c736e40a684b93e1a203403138baa976bf518d3db4884ffd4ca3e9a6L29) upstream and I haven't found a replacement.